src/fromager/build_environment.py

```diff
@@ -179,37 +178,44 @@ def _createenv(self) -> None:
             network_isolation=self._ctx.network_isolation,
         )
         logger.info("created build environment in %s", self.path)
+        if self._build_requirements:
+            self.install(self._build_requirements)
 
+    @property
+    def _pip_cmd(self) -> list[str]:
```

Comment:
Can we be more specific about the name like `_pip_install_cmd`  or am I overthinking this?

Reply:
That's reasonable feedback. 

@tiran I can make that edit in my branch, if you're OK with it?